public/sw.js

```diff
@@ -37,6 +43,12 @@ const fetchFromNetwork = (event) => {
 					}
 				})));
 			}
+
+			if (/avatar\/.*\?_dc/.test(event.request.url)) {
+				// handle the avatar updates
+				handleAvatar(event.request, response);
+			}
+
 			caches.open(version).then((cache) => cache.put(event.request, clonedResponse));
```

Comment:
@shubhsherl without return statement in the if it will put in cache twice? Please confirm

Reply:
It is required to update the avatars. RC gets any avatar update, with a param `_dc` which immediately updates the avatar.